src/plugins/data/public/now_provider/lib/get_force_now_from_url.ts

```diff
@@ -16,10 +16,25 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+
 import { parse } from 'query-string';
 
 /** @internal */
-export function parseQueryString() {
+export function getForceNowFromUrl(): Date | undefined {
```

Comment:
this moved from `timefilter` to this new `nowProvider`
Long term we should make apps responsible for getting this from the URL and setting it into `nowProvider`